### PR TITLE
Add feature importance processing algorithm

### LINKED ISSUE #64: Add feature importance processing algorithm

### PR DESCRIPTION
**Pull request by msorvoja**
Fixes #64 

PR https://github.com/GispoCoding/eis_toolkit/pull/468 in EIS Toolkit needs to be merged first.

### CODE REVIEW

File: eis_qgis_plugin/eis_processing/algorithms/exploratory_analysis/feature_importance.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        model_file_param.setHelp("The model file.")
+        self.addParameter(model_file_param)
+
+        evidence_data_param = QgsProcessingParameterMultipleLayers(
+            name=self.alg_parameters[1], description="Feature data", layerType=QgsProcessing.TypeRaster
+        )
+        evidence_data_param.setHelp(
+            "Data on which feature importance will be computed."
+        )
+        self.addParameter(evidence_data_param)
```
Comment (nmaarnio) [inline eis_qgis_plugin/eis_processing/algorithms/exploratory_analysis/feature_importance.py:45]: Could mention here that the data and order of rasters need to match the data used in training.

File: eis_qgis_plugin/eis_processing/algorithms/exploratory_analysis/feature_importance.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            name=self.alg_parameters[2], description="Target labels"
+        )
+        target_labels.setHelp("Target labels.")
+        self.addParameter(target_labels)
+
+        n_repeats_param = QgsProcessingParameterNumber(
+            name=self.alg_parameters[3],
+            description="N repeats",
+            type=QgsProcessingParameterNumber.Integer,
+            optional=True,
```
Comment (nmaarnio) [inline eis_qgis_plugin/eis_processing/algorithms/exploratory_analysis/feature_importance.py:57]: Not optional

File: eis_qgis_plugin/eis_processing/algorithms/exploratory_analysis/feature_importance.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            defaultValue=10
+        )
```
Comment (nmaarnio) [inline eis_qgis_plugin/eis_processing/algorithms/exploratory_analysis/feature_importance.py:59]: Should also define that minValue is 1

**Comment by msorvoja** [comment]
@nmaarnio Ready for another review.

**Pull request merged**